lib/Bindings/Python/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel",
+    "cmake>=3.12",
+    # MLIR build depends.
+    "numpy",
+    # Version 2.7.0 excluded: https://github.com/pybind/pybind11/issues/3136
+    "pybind11>=2.6.0,!=2.7.0",
```

Comment:
Can probably drop the exclusion now. I filed an issue with them and they pushed a dot version with the packaging fix.